cli/class-import.php

```diff
@@ -567,8 +568,8 @@ function( $file ) use ( $stable_path ) {
 	 *
 	 * @return string The plugin readme.txt or readme.md filename.
 	 */
-	protected function find_readme_file( $directory ) {
-		$files = Filesystem::list_files( $directory, false /* non-recursive */, '!^readme\.(txt|md)$!i' );
+	static function find_readme_file( $directory ) {
+		$files = Filesystem::list_files( $directory, false /* non-recursive */, '!(?:^|/)readme\.(txt|md)$!i' );
```

Comment:
I now figure that this change was probably mostly for consistency with the recursive searches added elsewhere.

Reply:
`list_files()` seemed to be matching against the full path, including the `$directory` prefix. With the `^` assertion only, it doesn't match any files. There might be something contextual here that's causing different behaviour, as otherwise `find_readme_file()` wouldn't work at all.

Reply:
Odd, it must be working at present though..